.sync.yml

```
@@ -0,0 +1,9 @@
+:global:
+  componentName: cluster-backup
+  githubUrl: https://github.com/projectsyn/component-cluster-backup
+  feature_goUnitTests: false
+  feature_goldenTests: true
+
+docs/antora.yml:
+  name: cluster-backup
+  title: cluster-backup
```

According to https://syn.tools/syn/how-tos/prepare_for_modulesync.html this should be a human friendly title
```suggestion
  title: Cluster Backup
```